ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/gmake/tasks/main.yml

```diff
@@ -10,7 +10,11 @@
 # Addresses https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/499
 # For RHEL 8, use gmake 4.2.1 already installed.
 
-- name: Test if self-built make 4.1 is available
+- name: Set make version
+  set_fact: makeVersion=4.1
+  tags: goodmake_source
+
+- name: Test if self-built make is available
```

Comment:
Would be nice to keep the version number in the ansible execution log so add {{ makeVersion }} into this

Reply:
AFAIK you can't use variables in ansible task names: https://stackoverflow.com/a/29001553/5838529

Reply:
actually I take that back, managed to make it work.